docker/Dockerfile

```diff
@@ -1,19 +1,16 @@
-# Creates a Docker image that is ready to execute tes compliance suite
+# Creates a Docker image that is ready to execute tes compliance suite                                                                                     Dockerfile                                                                                                 # Creates a Docker image that is ready to execute tes compliance suite
```

Comment:
Seems like there's a copy/paste issue here

Reply:
so this is really weird....if you just open the file, there is only 1 line but the PR does seem to show 2, not sure whats going on (github bug?)

https://github.com/elixir-cloud-aai/tes-compliance-suite/blob/cb698ccede6679d1f3347bf47f2bca6cdb8c308c/docker/Dockerfile (if I view file from here) 

or directly in my repo: https://github.com/yuliadub/tes-compliance-suite/blob/yuliadub/add_docker_file/docker/Dockerfile

Reply:
I think in all cases there is one line. But in the PR view it's wrapped, so it looks like two lines. I guess the issue is that the line is very long, with lots of white space in between.

Reply:
ah looking at this on my small laptop screen hid that from me, sorry about that - fixed!